ArduPlane/quadplane.cpp

```diff
@@ -416,6 +416,33 @@ const AP_Param::GroupInfo QuadPlane::var_info2[] = {
     // @Increment: .1
     // @User: Advanced
     AP_GROUPINFO("THROTTLE_EXPO", 10, QuadPlane, throttle_expo, 0.2),
+
+    // @Param: ACRO_ROLL_RATE
```

Comment:
should be ACRO_RLL_RATE

Reply:
thanks, ran into the character limit and forgot the comments